third_party/amd/lib/TritonAMDGPUToLLVM/Utility.cpp

```diff
@@ -2,17 +2,42 @@
 #include "PatternTritonGPUOpToLLVM.h"
 #include "mlir/Dialect/LLVMIR/NVVMDialect.h"
 #include "mlir/Dialect/LLVMIR/ROCDLDialect.h"
+#include "triton/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVMBase.h"
```

Comment:
Well, we shouldn't include this here, I'll move the needed functions to some common place :)